corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -449,10 +449,10 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
     });
 
     var GroupedQueryView = Marionette.CollectionView.extend({
-        tagName: "div",
+        tagName: "tr",
         template: _.template($("#query-view-group-template").html() || ""),
         childView: QueryView,
-        childViewContainer: "#query-properties",
+        childViewContainer: "#group-content",
```

Comment:
nit: It might be nice to have something more explicitly named since it's already the page already has some elements that use "content". Something like `query-group-content` maybe?

Reply:
sure thing